include/deal.II/base/time_stepping.templates.h

```diff
@@ -201,6 +209,20 @@ namespace TimeStepping
   // ----------------------------------------------------------------------
 
   template <typename VectorType>
+  ImplicitRungeKutta<VectorType>::ImplicitRungeKutta()
+    :
+    skip_linear_combi(false),
+    max_it(numbers::invalid_unsigned_int),
+    tolerance(-1.)
```

Comment:
Is it *required* that this value be initialized through a user call for everything to work? Or do things work in some sort of default way if the tolerance is `-1`? If it is *required* that a user sets the tolerance, then maybe initializing this variable with `numbers::signaling_NaN<double>()` makes more sense because it will yield an error upon first use.